packages/client/src/components/GrantsTable.spec.js

```diff
@@ -23,6 +23,7 @@ describe('GrantsTable component', () => {
         {
           interested_agencies: [],
           viewed_by_agencies: [],
+          followed_by_agencies: [],
```

Comment:
this component is very lightly tested and I'd like to return to add more test coverage here, but that can be done as a follow up PR; especially as we are likely removing some code soon for feature flag removal and therefore can reduce the test surface to cover